src/models/Expense.model.js

```diff
@@ -1,9 +1,43 @@
 'use strict';
 
 const { sequelize } = require('../db.js');
+const { DataTypes } = require('sequelize');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    spentAt: {
+      type: DataTypes.TEXT,
```

Comment:
Consider using a more appropriate data type for 'spentAt'. If this field is meant to store dates, using DataTypes.DATE might be more suitable than DataTypes.TEXT.